src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -378,26 +378,26 @@ Some components may not be hidden."</string>
     <string name="revanced_hide_navigation_create_button_title">Hide create button</string>
```

Comment:
I didn't capitalize "create" because the button doesn't have a label. IDK if this is correct tho 🤷

Reply:
In ReVanced, it seems to be written in capital letters
https://github.com/ReVanced/revanced-patches/blob/b72fe87cf9b0a5d8105ead9b50f0c72cf8cc4c97/src/main/resources/addresources/values/strings.xml#L429